### PR TITLE
Update agEUR, ANGLE tokens metadata

### PR DESCRIPTION
**Pull request by teddav**
I didn't have access to the Aurora address for both tokens previously, so this PR adds the correct address.

### CODE REVIEW

**Comment by teddav** [comment]
@sept-en should I also add the data in aurora.tokenlist.json (https://github.com/aurora-is-near/bridge-assets/blob/master/assets/aurora.tokenlist.json) ?

**Comment by teddav** [comment]
@sept-en I also added the metadata for both tokens to `assets/aurora.tokenlist.json`

File: assets/aurora.tokenlist.json [hunk 1]
```diff
@@ -55,6 +55,28 @@
...
+      "chainId": 1313161554,
+      "address": "0xb7e3617adb58dc34068522bd20cfe1660780b750",
+      "symbol": "ANGLE",
+      "name": "ANGLE",
+      "decimals": 18,
+      "logoURI": "https://raw.githubusercontent.com/aurora-is-near/bridge-assets/master/tokens/angle.svg",
+      "tags": [
+        "ethereum"
+      ]
+    },
```
Comment (sept-en) [inline assets/aurora.tokenlist.json:79]: Please don't modify that file, it's regenerating with the tool when the PR is merged

**Review by sept-en — COMMENTED** [review]
Thanks for your PR! Please fix my comment regarding the token list and should be good to go.

File: assets/aurora.tokenlist.json (hunk 1 above)
Comment (teddav) [reply assets/aurora.tokenlist.json]: ok, got it. sorry about that :)

**Comment by teddav** [comment]
@sept-en just fixed it. thanks for the comment!

**Review by sept-en — APPROVED** [review]

**Pull request merged**